manual-provision-payg/testsuite/start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# start.sh paygo provision.
+echo "Provisioning paygo..."
+sleep 2
+echo "Please wait."
+#udisksctl mount -b  /dev/sdb
+#mount unmounted removable devices
+udisksctl dump |
+awk -F':\n' -v'RS=\n\n' '/[ \t]*HintAuto:[ \t]*true/&&/\.Filesystem:/{
+                             print $1
+}' |
+while read dev
+do
+    udisksctl mount --object-path "${dev##*/UDisks2/}"
+done
+
+lsblk
+sleep 2
```

Comment:
What's this here for?

Reply:
gotten rid of it. it was left over after debug.